blog/2023-01-10-first-offsite/index.mdx

```diff
@@ -0,0 +1,123 @@
+
+So, we decided to try it out. After much planning, canceled bookings, and a few long flights later, our team met up in Santa Rosa, California.
+
+![offsite office](https://user-images.githubusercontent.com/52545545/211517211-ff1d5503-a367-4e6b-a670-067c447ed415.png)
+
+:::info
+My co-founder Jamil shares similar views about working together in person. He’s an even earlier pioneer in working remotely. I keep bugging him to write a blog about his 3 year journey living in a tiny house towed by his truck.
+
+For now, I’ll leave you with a link to his [slideshow](https://tinyhouse.jamilbk.com/).
+:::
+
+## What did we work on?
+
+Our itinerary included architecture planning, a multi-day hackathon, and plenty of fun activities. Let’s start with what we built.
+
+### REST API v0
+
+One common compliment from our users is how snappy and effortless it is to manage Firezone with the Web UI. Many IT teams support hundreds of concurrent users without a hitch. However, as fetching as our web portal is, we’d prefer if you rarely had to open it up. 
+
+Our REST API follows a series of updates to make Firezone easier to automate. This is why we made Docker the preferred (and soon-to-be only) method of deploying Firezone. And also why we’re building on the work from our community to release a first-party Terraform provider.
+
+You can take a look at the PR [here](https://github.com/firezone/firezone/pull/1155).
+
+### Splitting the gateway from the control plane
+
+Since we started the project a year ago, we’ve built features to make Firezone more suitable for larger teams. These organizations sometimes have complex network architectures, often spanning multiple cloud vendors and on-prem environments.
+
+The Firezone gateway is tightly coupled with the control plane. To support multi-site environments, we’ll need to support managing multiple gateways under a single control-plane. During the offsite, we architected and began work on this major change. 
+
+Take a look at the work in progress in [this PR](https://github.com/firezone/firezone/pull/1205). 
+
+### Documentation improvements
+
+We added additional docs on a few topics, including debug logs, split tunnel DNS considerations, and common troubleshooting issues.
+
+SEO was not top of mind when writing our docs, but we found ourselves part of the zero-sum game that appeases our Google overlords.
+
+Whether for goodwill or profit, you’ll find guides for Firezone of varying quality on the internet. Recently, we’ve noticed these ranking higher than our first-party docs.
+
+![google results](https://user-images.githubusercontent.com/52545545/211517741-4efb596a-3765-40d8-b483-f807f3a88e86.png)
+
+This is a problem because these guides are quickly outdated and often inaccurate. We know many prefer searching on Google instead of directly visiting our docs, so we made some updates to rank higher for popular searches.
+
+Take a look at the PR [here](https://github.com/firezone/firezone/pull/1194).
+
```

Comment:
This section feels more like a development / product update, and I fear it would be confused as an announcement of sorts. Think it might be possible to discuss "what we worked on" more generally? Could summarize this section.

Reply:
It does seem like a better fit for a general post on remote work to not focus too much on specifics only our user base cares about. I'll summarize and we can link other posts from this section once we write them (e.g. 0.7.0 launch).